Navitia io doc update

stifoon opened the pull request.

antoine-de committed:
Documentation: navitia io documentation update

update the navitia io doc from the v0 to v1

antoine-de committed:
documentation: update navitia doc

antoine-de committed:
Documentation: changes in the navitia io doc

antoine-de committed:
documentation: vincent review

Commit:
Documentation: navitia io doc update

places_nearby vs nearby_places
add "about the service"

Commit:
Documentation: add custom css

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
     :maxdepth: 1
```
Bon, je ne peux pas lâcher de comm' avant, mais la phrase :  At some point you will want to read: ne veut plus rien dire

Tristramg replied:
Tu dis ça juste parceque c’est moi qui l’ai écrit. C’est méchant.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
+    API Reference <http://navitia.io/docv1>
 
-Our APIs are available at the following url: http://api.navitia.io.
+There are no restrictions in using our API. However, please don't make more than one request per second.
```
On peut encore mettre ça ? Y a toujours rien de prévu à moyen terme ?

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
+********
 
-The region can be left out for requests that are based on coordinates. The format is one of json, xml or pb (for a result in Protocol Buffers).
+*navitia* is a RESTful API build on the `HATEOAS model <http://en.wikipedia.org/wiki/HATEOAS>`_. It returns `JSON <http://en.wikipedia.org/wiki/Json>`_ formated results.
```
Ouais, enfin HATEOAS, il faudrait corriger les liens avant.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
 
-.. warning::
-    This API is experimental. The parameters and responses are not definite as we will listen to your feedbacks to improve it.
+The latest API version is ``v1`` so the API url is http://api.navitia.io/v1.
```
On ne pourrait pas mettre directement, ce lien ? ça fait redite non ?

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
+The latest API version is ``v1`` so the API url is http://api.navitia.io/v1.
 
-    If you plan to build something successful, contact us to get an access with more vitamines and even more support.
+The API has been built on the `HATEOAS model <http://en.wikipedia.org/wiki/HATEOAS>`_ so the API should be quite self explanatory since the possible interactions are defined in hypermedia.
```
je trouve que ça fait pas mal de répétition, je pense qu'il ne faut pas le mettre au dessus.

l-vincent-l replied:
je pense que le possible n'est pas nécéssaire.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -34,26 +34,33 @@ At some point you will want to read:
+Have a look at the examples below to learn what API we provide and how to use them.
+
+.. note::
+    The results are paginated to avoid crashing your parser. The parameters to get the next or previous page are within the ``links`` section of the result.
```
Je ne suis pas génial en anglais, mais c'est pas in à la place de within ?

antoine-de replied:
je pense que within est correct la.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -63,92 +70,121 @@ This chapter shows some usages with the minimal required arguments. However, thi
 **************************
 
-Let's find out how to get from the view point of the Golden Gate bridge to the Transamerica Pyramid in San Francisco the 18th of April at 08:00 AM.
+Let's find out how to get from the view point of the Golden Gate bridge to the Transamerica Pyramid in San Francisco the 18th of January at 08:00 AM.
```
On retombe sur le problème qu'on a toujours eu ici, soit on change la documentation tous les mois soit, on versionne aussi le build et on met un peu de JS, soit on met une instance de test qui aurait toujours les mêmes données.

antoine-de replied:
oui complètement d'accord. A mon avis ca peut etre géré en JS avec la refonte de navitia.io

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -63,92 +70,121 @@ This chapter shows some usages with the minimal required arguments. However, thi
-* ``destination=coord:-122.402770:37.794682``
+* ``from=-122.4752;37.80826``
+* ``to=-122.402770;37.794682``
 * ``datetime=20130418T0800``
```
Il faut changer la date.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -63,92 +70,121 @@ This chapter shows some usages with the minimal required arguments. However, thi
-If you are wondering why the origin and destination have such a syntax, it's because we allow
-to provide an address or a specific station as input. But we will see more about that later in the
-section about entry points.
+In the ``links`` section there is for example this link: ``"href": "http://api.navitia.io/v1/coverage/{regions.id}/lines"``
```
J'aurai un peu plus détaillé ici : 
Peut être parler de l'attribut rel qui donne l'objet en lien, et aussi de l'argument templated qui permet de savoir si l'url en templated ou non.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -63,92 +70,121 @@ This chapter shows some usages with the minimal required arguments. However, thi
 The ``places`` API finds any object whose name matches the first letters of the query.
 
-http://api.navitia.io/v0/sf/places.json?q=trans
+To find the objects that start with "trans" the request should be: http://api.navitia.io/v1/coverage/ny/places?q=tran
```
il manque le s de trans

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -63,92 +70,121 @@ This chapter shows some usages with the minimal required arguments. However, thi
 
-The ``nearby_places`` API finds any object within a certain radius as a crow flies.
+The ``places_nearby`` API finds any object within a certain radius as a crow flies.
+This API is not accessible from the main endpoint but has to be applied on a stop point, an address, a coordinate,...
```
Je pense que coordinates est pluriel.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -157,6 +193,19 @@ At last, we are present on IRC on the network Freenode, channel #navitia.
+.. warning::
+  Authentication is not required now, but will soon become (within april 2014). 
+  Please contact us for details.
+  And remember that there are no restrictions in using our API :)
```
ça vient en contradiction avec ce qui est dit au départ.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -157,6 +193,19 @@ At last, we are present on IRC on the network Freenode, channel #navitia.
 
+About the service
+*****************
+This API is experimental. The parameters and responses are not definite as we will listen to your feedbacks to improve it.
```
Encore ? Je pense que le média va apprécier.

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -164,6 +213,8 @@ The street network is extracted from `OpenStreetMap <http://www.openstreetmap.or
 
 The public transport data are provided by networks that provide their timetables as open data.
 
+Some data upgrade are achieved by Canal TP.
```
je ne suis pas certain que upgrade soit approprié peut être plus improvment

l-vincent-l commented on documentation/navitia.io/source/index.rst:
```diff
@@ -157,6 +193,19 @@ At last, we are present on IRC on the network Freenode, channel #navitia.
+If you plan to build something successful, contact us to get an access with more vitamins and even more support.
+
+.. warning::
+  Authentication is not required now, but will soon become (within april 2014). 
```
On fait des paris sur les dates ?

l-vincent-l commented:
Et pour avoir un site cohérent il faut aussi inclure la documentation d'intégration.

antoine-de committed:
documentation: merge integration doc with navitia io

merge api reference with the navitia io doc.

update the doc with Vincent review

antoine-de committed:
documentation: css change for page width

the width was too small. changed it to a standard 960px

l-vincent-l commented:
Il faudrait checker que les liens sont bons dans la documentation de référence, je pense en particulier aux datetimes

Commit:
Documentation: Correction sur les liens date_time
